### DISCUSSION TITLE
Should Filament assets be moved into the "vendor" folder?

### DISCUSSION THREAD

**Discussion by SupianIDz** [filamentphp/filament#8415] [Help]
Why are the default assets from the filament not put in the vendor folder like in the general package? I see that Filament's default CSS and JS files are minified, and I don't think there's any reason to hard code them.

If there is a reason for that, please tell me.

**Comment by danharrin** [comment]
Vendor files are not accessible in the browser. They are copied into /public so they are public.

**Reply by SupianIDz** [reply]
you are right, I mean the assets that come from the package are put in the vendor folder located in the public path like `/public/vendor/filament/js` and `/public/vendor/filament/css` instead of in the `/public/css/filament` and `/public/js/filament` folders.

Even using official Laravel implements what I mean https://github.com/laravel/telescope/blob/4.x/src/TelescopeServiceProvider.php#L92-L94
https://github.com/laravel/horizon/blob/5.x/src/HorizonServiceProvider.php#L80-L82
https://github.com/laravel/vapor-ui/blob/master/src/VaporUiServiceProvider.php#L34-L36

I think this is worth considering grouping assets from filament with assets from other packages in the `/public/vendor` folder,

from this
<img width="452" alt="Screenshot 2023-09-12 at 12 02 25" src="https://github.com/filamentphp/filament/assets/37969970/80059ef7-1588-4d3b-a6f1-bd162cada621">

to this
<img width="463" alt="Screenshot 2023-09-12 at 12 03 47" src="https://github.com/filamentphp/filament/assets/37969970/6371fcd3-8823-47cd-8c65-cb3c11fe5e0e">



Sorry for the lack of clarity

**Reply by danharrin** [reply]
Honestly it doesnt really matter where the files are, right? But we could make it configurable from config/filament.php if you want to PR that.
